tools/clang/test/SemaHLSL/getattribute-at-vertex.missing.nointerpolation.call.hlsl

```diff
@@ -0,0 +1,11 @@
+// RUN: dxc -T ps_6_2 -E main %s -spirv -verify
```

Comment:
The SemaHLSL tests need to run on both spirv and dxil backends. Could you please remove the -spirv argument?

Reply:
I see that this is a sema check specific to SPIR-V. In that case it would probably be best to have these tests in a subfolder with `lit.local.cfg` file that would have `config.unsupported = not config.spirv`.

Reply:
We can solve this by putting `// REQUIRES: spirv` in the test. That will make it only run if the spir-v code generator is enabled.

Reply:
There is one thing I don't get:
If we never add tests with `-spirv`, like in https://github.com/microsoft/DirectXShaderCompiler/pull/6467, we essentially lose coverage for the whole SPIR-V compilation side no?

As in, today, SEMA is shared, but if one day something goes bad, and for some reason sema diagnostics don't show up with `-spirv`, none of those tests would catch it.
Shouldn't all the sema tests also have a run line with `-spirv`?

Reply:
Counter point: If you look at clang tests for parsing and semantic analysis of C++ code don't have separate run lines for every target that LLVM supports.

Adding loads of duplicated test coverage that provides no additional meaningful test coverage does nothing but slow down development.

Reply:
I see, yes, that would be unscalable. Thanks!

Reply:
Added the `// REQUIRED: spirv` to the 2 impacted SEMA tests.